plugins/git/README.md

```diff
@@ -205,9 +206,8 @@ These are aliases that have been removed, renamed, or otherwise modified in a wa
 | ggpull | `git pull origin $(current_branch)`                    | new alias `ggl` (`ggpull` still exists for now though) |
 | ggpur  | `git pull --rebase origin $(current_branch)`           | new alias `ggu` (`ggpur` still exists for now though)  |
 | ggpush | `git push origin $(current_branch)`                    | new alias `ggp` (`ggpush` still exists for now though) |
-| gk     | `gitk --all --branches`                                | now aliased to `gitk --all --branches`                 |
```

Comment:
there was no change for `gk` here that I could find via git blame. It always pointed to `gitk --all --branches`